ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2-nonprod.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
```

Comment:
Please change file name to "NotificationEvents-SDO-R2-nonprod.json" so its quickly identifiable when we do release of this feature.